modules/creating-project-scoped-resources-for-your-project.adoc

```diff
@@ -0,0 +1,123 @@
+endif::[]
+ifndef::upstream[]
+Cluster administrators can create project-scoped resources for any project, as described in link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/creating-project-scoped-resources_managing-rhoai[Creating project-scoped resources].
+endif::[]
+====
+
+.Prerequisites
+
+* You can access the {openshift-platform} console.
+
+ifdef::upstream[]
+* An {productname-short} administrator has set the `disableProjectScoped` dashboard configuration option to `false`, as described in link:{odhdocshome}/managing-resources/#customizing-the-dashboard[Customizing the dashboard].
+endif::[]
+ifndef::upstream[]
+* An {productname-short} administrator has set the `disableProjectScoped` dashboard configuration option to `false`, as described in link:{rhoaidocshome}{default-format-url}/managing_resources/customizing-the-dashboard[Customizing the dashboard].
+endif::[]
+
+* You can access a project in the {productname-short} console.
+
+* You have example YAML code for the type of resource that you want to create.
++
+You can get the YAML code from a trusted source, such as an existing project-scoped resource, a Git repository, or documentation.
+Alternatively, you can contact your cluster administrator to request the relevant YAML code.
+
+
+.Procedure
+. Log in to the {openshift-platform} console.
+. Copy the YAML code from a trusted source to create the resource.
++
+Alternatively, if you can access an existing project-scoped resource in one of your projects, you can copy the YAML code as follows:
+
+.. In the *Administrator* perspective, click *Home* -> *Search*.
+.. From the *Projects* list, select the appropriate value.
+.. In the *Resources* list, search for the relevant resource type, as follows:
+
+* For workbench images, search for `ImageStream`.
+// When RHOAIENG-26758 is resolved, remove this line and uncomment the following two lines.
+// * For hardware profiles, search for `HardwareProfile`.
+// * For accelerator profiles, search for `AcceleratorProfile`.
+* For serving runtimes, search for `Template`. 
+From the resulting list, find the templates that have the `objects.kind` specification set to `ServingRuntime`. 
+
+.. Select a resource, and click the *YAML* tab. 
+.. Copy the YAML content, and click *Cancel*.
+
+. From the *Project* list, select your project name. Note the spelling.
+. From the toolbar, click the *+* icon to open the *Import YAML* page.
+. Paste the example YAML content into the code area.
+. Edit the `metadata.namespace` value to specify the name of your project.
+. If necessary, edit the `metadata.name` value to ensure that the resource name is unique within the specified project.
+. Optional: Edit the resource name that is displayed in the {productname-short} console, as follows:
+
+* For workbench images, edit the `metadata.annotations.opendatahub.io/notebook-image-name` value.
```

Comment:
FYI, editing the `metadata.name` is required only if there is an existing resource with the same name in the same namespace. `displayName` can be edited but only for the user's recognition. Otherwise if the user has 2 resources in global as well as project-scoped namespace, they will find the same names in the dropdown in both the project-scoped and global sections.

Reply:
I think your points are already covered in the existing text? Let me know if you want to change anything.